src/System.Runtime.Extensions/tests/System/Environment.Exit.cs

```diff
@@ -42,6 +42,15 @@ public static void CheckExitCode(int expectedExitCode)
 
         [Theory]
         [MemberData(nameof(ExitCodeValues))]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapNotUapAot, "RemoteExecutor behaves different in UAP.")]
```

Comment:
Why does this need its own test?  i.e. why can't the existing CheckExitCode test just be modified to use ExpectedExitCode?